run-tests

```diff
@@ -19,6 +19,17 @@ if sudo --non-interactive "$name/ci/lvm-manage" setup-lvm vg$$/pool; then
     CLEANUP_LVM=yes
 fi
 
+CLEANUP_KERNEL_POOL=
+if [ ! -d "/var/lib/qubes/vm-kernels" ]; then
```

Comment:
This won't work when running tests in actual dom0 (vm-kernels dir exists, but "1.0" kernel doesn't). 
Some of the tests patch `qubes_kernels_base_dir`, see for example `test_260_kernelopts` in `qubes/tests/vm/qubesvm.py` which should be safer approach.

Reply:
Yes. Just recognized it. I will do the tests locally in order to avoid abusing the CI/CD servers and cluttering the logs. Will come back after I fix it properly

Reply:
@marmarek I finally managed to patch one of the major base classes (`AdminAPITestCase`). I believe that I can go forward and properly patch the rest.

I have a question regarding the CI/CD servers. Are they owned by Qubes project or they belong to Gitlab? I am trying to avoid using the online ones as much as possible and test the patches locally. But using the online ones are sometimes more convenient.

Reply:
> I have a question regarding the CI/CD servers. Are they owned by Qubes project or they belong to Gitlab? I am trying to avoid using the online ones as much as possible and test the patches locally. But using the online ones are sometimes more convenient.

We use our own runners, you can use them as much as you like :)